packages/idealo.design/src/ui/pages/ComponentsPage/component-consumer-contract.spec.js

```diff
@@ -0,0 +1,100 @@
+import {Pact, Matchers} from '@pact-foundation/pact'
+import {
+    fetchMap,
+    fetchTags,
+    fetchComponents
+} from './component_data'
+import path from 'path'
+
+const {eachLike, like} = Matchers
+
+const PORT = 4000;
+const URL = 'http://localhost:';
+
+const provider = new Pact({
+    consumer: 'ComponentConsumer',
+    provider: 'ComponentProvider',
+    port: PORT,
+    log: path.resolve(process.cwd(), 'src/__tests__/pact', 'pact.log'),
+    dir: path.resolve(process.cwd(), 'src/__tests__/pact'),
+    logLevel: 'INFO',
+});
+
+export const mockupComponent = {
+    "component_id": 3,
+    "title": "@motif/checkbox"
+}
+
+export const mockupTags = {"tag_name": "button"}
+
+export const mockupMap = {
+    "component_id": 4,
+    "title": "@motif/button",
+    "tag_name": "button"
+}
+
+describe('all Tests', () => {
+    afterAll(() => provider.finalize());
+    beforeAll(() => provider.setup());
+    afterEach(() => provider.removeInteractions())
+
+    describe('test Components+Tags(Map) list', () => {
```

Comment:
Not sure if we need to prefix the description with test. I do not see a need for it.